staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -492,6 +496,11 @@ func (o *ApplyOptions) applyOneObject(info *resource.Info) error {
 		WithFieldManager(o.FieldManager).
 		WithFieldValidation(o.ValidationDirective)
 
+	lastAppliedHelper := resource.NewHelper(info.Client, info.Mapping).
+		DryRun(o.DryRunStrategy == cmdutil.DryRunServer).
+		WithFieldManager(FieldManagerClientSideApply).
```

Comment:
I haven't read the whole thing yes, but that sounds like the manager you'll use to apply the last applied annotation, didn't we agree to use a different field manager for that?

Reply:
It is a different field manager since its operation is `Apply` instead of `Update` :). Happy to change if that isnt convincing enough